src/System.Net.HttpListener/tests/HttpListenerWebSocketTests.cs

```diff
@@ -35,7 +35,7 @@ public void Dispose()
         }
 
         [ActiveIssue(22164, TargetFrameworkMonikers.Uap)] // Fails in RS2 only (works in RS3).
-        [ConditionalTheory(nameof(IsNotWindows7))]
+        [ConditionalTheory(nameof(PlatformDetection.ClientWebSocketPartialMessagesSupported), nameof(IsNotWindows7))]
```

Comment:
Actually we need the `nameof(IsNotWindows7)`.

`bool ClientWebSocketPartialMessagesSupported => !IsUap || IsWindows10InsiderPreviewBuild16215OrGreater;`
It's a OR logic, and ( Win7 && !uap) will return true.